docs/releases/v2018.1.rst

```diff
@@ -0,0 +1,284 @@
+
+ar71xx-nand
+===========
+
+* ZyXEL
+
+  - NBG6716
+
+ar71xx-tiny
+===========
+
+* TP-Link
+
+  - TL-WA901ND v5
+
+ipq806x [#newtarget]_ [#noibss]_
+================================
+
+* TP-Link
+
+  - Archer C2600
+
+ramips-mt7620 [#newtarget]_ [#noibss]_
+======================================
+
+* GL Innovations
+
+  - GL-MT300A
+  - GL-MT300N
+  - GL-MT750
+
+ramips-mt7628 [#newtarget]_ [#noibss]_
+======================================
+
+* VoCore
+
+  - VoCore 2
+
+ramips-rt305x [#newtarget]_ [#noibss]_
+======================================
+
+* A5
+
+  - V11
+
+* D-Link
+
+  - DIR615 (D1, D2, D3, D4, H1)
+
+* VoCore
+
+  - VoCore (8MB, 16MB)
+
+sunxi [#newtarget]_
+===================
+
+* LeMaker
+
+  - Lamobo r1
+
+
+.. [#newtarget]
+  New target
+
+.. [#noibss]
+  Device or target does not support AP+IBSS mode: This device or target will not be built
+  when *GLUON_WLAN_MESH* is set to ``ibss``.
+
+New features
+************
+
+Multidomain support
+===================
+
+When mesh networks grow too large, is it often necessary to split them into
+multiple independent mesh domains to allow the meshes to work with acceptable
+performance. Formerly, the only way to achieve this with Gluon was to build
+a separate set of firmware images for each domain.
+
+With Gluon 2018.1, multidomain firmwares can be used to achieve the same,
+using only a single site configuration, with only short snippets of domain-specific
+configuration. The feature is explained in detail in :doc:`../features/multidomain`.
+
+Wired mesh encapsulation
+========================
+
+Gluon now supports encapsulating wired mesh traffic (Mesh on LAN/WAN) in
+`VXLAN <https://en.wikipedia.org/wiki/Virtual_Extensible_LAN>`_, a new standard with
+usecases similar to VLANs, but a much greater ID space of 24bit; in addition, VXLAN
+packets pass through VLAN-aware switches without any special configuration.
+
+Encapsulating mesh traffic has two advantages:
+
+* By using a different VXLAN ID for each site and mesh domain, accidental
+  wired mesh connections between nodes of different domains are not possible anymore.
+  This is especially important when nodes switch mesh domains automatically, as it
+  is done in some sites (using site-specific packages).
+* While batman-adv traffic does not interact with non-mesh traffic in the same wired
+  network in any way (so Gluon nodes can mesh over existing wired networks), this is
+  not the case for layer 3 mesh protocols like Babel. Encapsulating the traffic allows
+  to distinguish mesh traffic from unrelated packets.
+
+As enabling VXLAN encapsulation will prevent wired mesh communiacation with old nodes
+that do not support VXLAN yet, VXLANs can be enabled per-domain using the site configuration
+setting *mesh.vxlan*. VXLAN is enabled by default in multidomain setups; in single-domain
+site configurations, the *mesh.vxlan* setting is mandatory. We recommend to enable
+VXLAN encapsulation in all new sites and domains.
```

Comment:
do we want to announce some kind of deprecation timeframe for the old cable-based meshing type?

Reply:
Something like "when we have a robust migration path, it will be removed in the next major release". I don't think we can give a more concrete timeframe at this point (same for IBSS).

Reply:
yes, something like this

Reply:
Done